.github/workflows/acquia_cms_ci.workflow.yml

```diff
@@ -330,10 +351,13 @@ jobs:
         orca-job:
           - INTEGRATED_TEST_ON_LATEST_LTS
           # - INTEGRATED_TEST_ON_PREVIOUS_MINOR
-          # - ISOLATED_TEST_ON_CURRENT
+          - ISOLATED_TEST_ON_CURRENT
         php-version:
           - 8.1
           - 8.3
+        exclude:
+          - php-version: 8.1
```

Comment:
@rajeshreeputra Instead of this we can only add 8.3 version as per https://www.drupal.org/docs/getting-started/system-requirements/php-requirements

Reason we are only supporting D10 and 11

cc: @chandan-singh7929 @vishalkhode1

Reply:
No, let's not drop PHP 8.1 support.